<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="opencloseButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open/ Close</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="receiTextEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>351</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sendTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="statuslabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="comportlabel">
        <property name="text">
         <string>COM Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comportBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="baudratelabel">
        <property name="text">
         <string>BaudRate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudrateBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="databitslabel">
        <property name="text">
         <string>DataBits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databitsBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="paritylabel">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="parityBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="stopbitslabel">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopbitsBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="timeoutlabel">
        <property name="text">
         <string>Time Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeoutspinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="querymodelabel">
        <property name="text">
         <string>QueryMode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="querymodeBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
